hashintel/hash
# H-356: Handle inheritance in the entity editor and entities tab

## 🌟 What is the purpose of this PR?

#2909 allows for setting parents of types, so that a given type inherits properties/links from its parents.

This PR handles showing those inherited properties/links in the (1) entity editor and (2) entities tab listing the entities of a given type, on its type page.

## 🚫 Blocked by

- [x] Merging of #2909 

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ❓ How to test this?



1.  Try creating an entity of a type with parents
2. Check it has parent properties available
3. Check its display on the entities tab

## 📹 Demo



https://github.com/hashintel/hash/assets/37743469/44a5e4f8-e04d-4f3a-b227-5ece9aebe124
